dracut/modules.d/99kiwi-lib/kiwi-lib.sh

```diff
@@ -10,6 +10,11 @@ function setup_debug {
     fi
 }
 
+function kiwi_die {
+    echo 7 > /proc/sys/kernel/printk
+    die $@
```

Comment:
`shellcheck` fails for this line with:

`SC2068: Double quote array expansions to avoid re-splitting elements.`

Reply:
pushed an amended version